requirements.txt

```diff
@@ -8,4 +8,8 @@ hnswlib
 xxhash
 tenacity
 dspy-ai
-neo4j
\ No newline at end of file
+neo4j
+pymilvus
```

Comment:
Remove the milvus dep, so those who want to use this component will handle the cross platform. Let's not introduce the install problems for the rest.